providers/aws/resources/aws_config.go

```diff
@@ -73,12 +73,17 @@ func (a *mqlAwsConfig) getRecorders(conn *connection.AwsConnection) []*jobpool.J
 					recording = val.recording
 					lastStatus = val.lastStatus
 				}
+				resourceTypesInterface := make([]interface{}, len(r.RecordingGroup.ResourceTypes))
+				for i, resourceType := range r.RecordingGroup.ResourceTypes {
+					resourceTypesInterface[i] = string(resourceType)
+				}
 				mqlRecorder, err := CreateResource(a.MqlRuntime, "aws.config.recorder",
 					map[string]*llx.RawData{
 						"name":                       llx.StringDataPtr(r.Name),
 						"roleArn":                    llx.StringDataPtr(r.RoleARN),
 						"allSupported":               llx.BoolData(r.RecordingGroup.AllSupported),
 						"includeGlobalResourceTypes": llx.BoolData(r.RecordingGroup.IncludeGlobalResourceTypes),
+						"resourceTypes":              llx.ArrayData(resourceTypesInterface, types.Any),
```

Comment:
According to your definition in the lr file, you should use `types.String` instead of `types.Any`.

Reply:
Thanks Christian. Done